images/openshift-enterprise-pod.yml

```diff
@@ -1,15 +1,11 @@
-container_yaml:
-  go:
-    modules:
-    - module: github.com/openshift/images/pod
-      path: pod
```

Comment:
It's still built with golang, we'll still want source analysis so I think this should still be configured. Perhaps
```
container_yaml:
￼  go:
￼    modules:
￼    - module: github.com/openshift/kubernetes/build/pause
```
? (I'm not sure what exactly is considered the module here)

Reply:
No, it's plain C, built with gcc.

Reply:
Yup, this is all C, there is no Go involved in the build process.

Reply:
ah, i thought because it used the golang builder, it was golang. but the builder also has gcc. thanks for the correction.